### PR TITLE
frontend/feat: #2872 added accessibility assets

### PR DESCRIPTION
**Pull request by MercyQueen**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [x] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

### CODE REVIEW

**Review by kavya-shree-s — APPROVED** [review]

File: Frontend/android-native/app/src/user/common/res/drawable/carousel_4.png
Comment (witcher-shailesh) [inline Frontend/android-native/app/src/user/common/res/drawable/carousel_4.png:1]: reduce the size, compress and then upload. Check with @Vignesh-772 / @Rohit4997  for compressing while retaining quality

File: Frontend/android-native/app/src/user/common/res/drawable/ny_ic_accessibility_banner_img.png
Comment (witcher-shailesh) [inline Frontend/android-native/app/src/user/common/res/drawable/ny_ic_accessibility_banner_img.png:1]: same here

File: Frontend/android-native/app/src/user/common/res/drawable/ny_ic_blind_pickup.png
Comment (witcher-shailesh) [inline Frontend/android-native/app/src/user/common/res/drawable/ny_ic_blind_pickup.png:1]: reduce size

**Review by witcher-shailesh — CHANGES_REQUESTED** [review]

**Comment by witcher-shailesh** [comment]
@kavya-shree-s make sure images are compressed and not big in size

File: Frontend/android-native/app/src/user/common/res/drawable/carousel_4.png
Comment (MercyQueen) [reply Frontend/android-native/app/src/user/common/res/drawable/carousel_4.png:1]: Updated.

**Review by witcher-shailesh — APPROVED** [review]

**Pull request merged**